extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -81,21 +203,51 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
 
     /** @private {?Element} */
     this.ampAnalyticsElement_ = null;
+
+    /** @private {?Object<string,*>}*/
+    this.jsonTargeting_ = null;
+
+    /** @private {number} */
+    this.adKey_ = 0;
+
+    // TODO(keithwrightbos) - how can pub enable?
+    /** @private @const {boolean} */
+    this.useSra_ = getMode().localDev && /(\?|&)force_sra=true(&|$)/.test(
```

Comment:
Given the size of this PR, I planned to add experiment code in another PR once this is merged